identifiers/country-fr/oversea.csv

```diff
@@ -0,0 +1,12 @@
+id,name
+ocd-division/country:fr/oversea_collectivity:mf,St-Barthélemy/St-Martin
```

Comment:
The correct term is "oversea**s** collectivity" not "oversea_collectivity". Please use `overseas_collectivity` and update the readme.

Reply:
Similarly, please correct `oversea_department` to `overseas_department`.

Reply:
I corrected both.